### PR TITLE
Add openmetricstest parse positive and negative test cases

### PR DESCRIPTION
**Pull request by robskillington**
This adds positive and negative parse test cases from the Prometheus python client to openmetricstest and a helper to run the test cases against the Prometheus python client using docker.

### CODE REVIEW

File: tests/testdata/parsers/no_newline_after_eof/metrics.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# TYPE a gauge
+# HELP a help
+a 1
+# EOF
```
Comment (brian-brazil) [inline tests/testdata/parsers/no_newline_after_eof/metrics.txt:4]: Are you sure there's no newline here?

**Review by brian-brazil — COMMENTED** [review]
This looks to be missing the roundtrip, plus https://github.com/prometheus/client_python/pull/602

CI is also not happy.

File: tests/testdata/parsers/no_newline_after_eof/metrics.txt (hunk 1 above)
Comment (robskillington) [reply tests/testdata/parsers/no_newline_after_eof/metrics.txt:4]: Looks good:
```
$ xxd tests/testdata/parsers/no_newline_after_eof/metrics.txt
00000000: 2320 5459 5045 2061 2067 6175 6765 0a23  # TYPE a gauge.#
00000010: 2048 454c 5020 6120 6865 6c70 0a61 2031   HELP a help.a 1
00000020: 0a23 2045 4f46                           .# EOF
$ xxd tests/testdata/parsers/simple_counter/metrics.txt
00000000: 2320 5459 5045 2061 2063 6f75 6e74 6572  # TYPE a counter
00000010: 0a23 2048 454c 5020 6120 6865 6c70 0a61  .# HELP a help.a
00000020: 5f74 6f74 616c 2031 0a23 2045 4f46 0a    _total 1.# EOF.
```

**Comment by robskillington** [comment]
Added the roundtrip test and updated with latest changes from https://github.com/prometheus/client_python/pull/602 @brian-brazil 👍

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**